pipelines/deploy-release/deploy-release.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: deploy-release
   labels:
-    app.kubernetes.io/version: "0.13.0"
+    app.kubernetes.io/version: "0.13.1"
```

Comment:
Shouldn't we be upversioning the minor version (so 0.14.0 here)? The micro should only be used for bug fixes I think. But I don't really mind, just saying ;)

Reply:
The ticket for this is RHTAPBUGS-1077, so by nature I considered this a bug fix

Reply:
I see, I didn't realize that. OK then.